src/validators/daterange.js

```diff
@@ -19,11 +19,11 @@ export default class DateRangeValidator {
    * Validates the date ranges
    */
   isValid() {
-    if (!new DateControlValidator(this.from || {}).isValid()) {
+    if (!new DateControlValidator(this.from || {}).validDate()) {
```

Comment:
This `|| {}` is redundant as the `DateControlValidator` will already default to an empty object when `this.from` is undefined. This can technically cause an issue if `this.from` is set to `false` or `null`, but since it is being controlled from within the component I think we can be sure it isn't.

Reply:
Yeah, I'll try and start removing these as I come across them if it's safe. Updated https://github.com/18F/e-QIP-prototype/pull/1026/commits/1210c9202e4cc2b869858085061f2782b4f5cad0

Reply:
So I guess in this case these were needed to support `null` values being passed from tests. I updated the tests to pass in a `{}` instead, but I wonder if that is part of the explanation for why these are all around?